lib/improver/utilities/cube_manipulation.py

```diff
@@ -260,7 +267,7 @@ def equalise_cubes(cubes_in, merging=True):
     cubes = iris.cube.CubeList([])
     for cube in cubes_in:
         cubes.append(cube.copy())
-    _equalise_cube_attributes(cubes)
+    _equalise_cube_attributes(cubes, model_id_attr)
```

Comment:
We shouldn't rely on position for a keyword attribute.
```suggestion
    _equalise_cube_attributes(cubes, model_id_attr=model_id_attr)
```

Reply:
Done.